parlai/crowdsourcing/tasks/fast_acute/README.md

```diff
@@ -0,0 +1,71 @@
+# *** TODO: rewrite all this! ***
```

Comment:
Leaving a quick comment to point out the `todo`s floating about. Assuming there's still a few things to be fixed up here? :)